README.md

```diff
@@ -9,6 +9,10 @@ Planning Competitions (IPC).
 * Build single generator: ``cd assembly; make``
 * Test generators: ``sudo apt install python-tox && tox``
 
+## Docker instructions
+* Build in Docker image: ``docker build -t pddl-generators .``
+* Run Docker container: ``docker run -ti -t pddl-generators``
```

Comment:
Double `-t`?

Reply:
I think it would be good to use long option names for Docker beginners: --tag, --tty, --interactive